src/main/java/de/tum/in/www1/artemis/config/CacheConfiguration.java

```diff
@@ -39,6 +40,7 @@
 import tech.jhipster.config.JHipsterProperties;
 import tech.jhipster.config.cache.PrefixedKeyGenerator;
 
+@Profile({ PROFILE_CORE, "buildagent" })
```

Comment:
why don't you use PROFILE_BUILDAGENT instead of "buildagent"?

Reply:
na gut